ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```diff
@@ -78,6 +78,39 @@ keys:
                     description: |
                       Supported options are platform dependent
                       Example: "< rate > kbps", "1-100 percent", "< rate > pps"
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: |
+                          Turn on random-detect ecn count
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            type: str
+                            valid_values: ["segments", "bytes", "kbytes", "mbytes"]
+                          min:
```

Comment:
Please look into improving the schema with `min` and `max` values if we know them for all keys in this area.
Also to catch some corner cases with YAML input and inline jinja, please add
```yaml
convert_types:
  - str
```
under all `type: int` keys.

Reply:
So because the units can be different, we cannot set a min/max. But still add the convert_types.

Reply:
I set the max to the absolute maximum if the smallest unit is used with the platform that has the largest buffers. Shall I keep that or remove it altogether ? 
We know the min is 1, so I can leave it there.

Reply:
Maybe it will be better to remove the limits since it might be misleading if someone thinks they can use the max no matter which unit.

Reply:
Removed the max limit but left the minimum.